test/integration/client-side-encryption/client_side_encryption.spec.test.ts

```diff
@@ -109,6 +109,13 @@ describe('Client Side Encryption (Legacy)', function () {
       if (typeof result === 'string') return result;
     }
 
+    if (['Insert with deterministic encryption, then find it'].includes(description)) {
```

Comment:
I noticed this test never runs because there is another test with the same name in a different suite that is in the skippedAuthTests list.  We probably need to update our skip logic to account for multiple suites, or we need to unskip the other test (maybe it'll pass now, who knows).  

Regardless, can you adjust the PR to make sure that all the new tests are running and passing?